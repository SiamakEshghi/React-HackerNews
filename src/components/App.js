import React, {Fragment} from 'react';
import Header from 'components/Header';

export default ({ children }) => {
    return (
        <Fragment>
            <Header />
            {children}
        </Fragment>
    );
};
