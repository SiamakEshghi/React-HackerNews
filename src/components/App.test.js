import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from 'components/App';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<App />);
});

it('Ahould render correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
});