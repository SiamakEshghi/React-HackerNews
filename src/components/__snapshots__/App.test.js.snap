// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ahould render correctly 1`] = `
<Fragment>
  <Component />
</Fragment>
`;
